scripts/globals/znm.lua

```diff
@@ -57,86 +153,94 @@ xi.znm.soultrapper.getZeniValue = function(target, user, item)
     local zeni = 5
 
     -- HP% Component
-    local hpMultiplier = math.min(100 / hpp, 5)
-    if hpp <= 5 then
-        hpMultiplier = 10
-    end
-    zeni = zeni * hpMultiplier
-
-    -- In-Demand System Component
-    -- TODO: Make rotating server var
-    local inDemandSystem = xi.ecosystem.DRAGON
-    if system == inDemandSystem then
-        zeni = zeni * 1.5
-    end
-
-    -- NM/Rarity Component
-    if isNM then
-        zeni = zeni * 1.5
-    end
+    zeni = zeni * xi.znm.SOULPLATE_HPP_MULT * math.exp(100-hpp)
 
     -- Distance Component
-    zeni = zeni * utils.clamp((1 / distance) * 2, 1, 2)
+    zeni = zeni * utils.clamp((1 / distance) * 8, 1, 2)
 
     -- Angle/Facing Component
     if isFacing then
-        zeni = zeni * 1.5
+        zeni = zeni * xi.znm.SOULPLATE_FACING_MULT
     end
 
     -- Bonus for HS Soul Plate
     if user:getEquipID(xi.slot.AMMO) == xi.items.BLANK_HIGH_SPEED_SOUL_PLATE then
-        zeni = zeni * 1.5
+        zeni = zeni * xi.znm.SOULPLATE_HS_MULT
```

Comment:
I dont know if this changed from 75 era to now retail - if so, please disregard

I dont see anything that takes into account:
- Mob level -  can take a pic of a level 1 mob as a 75 with no penalty.  In 75 era taking a picture of EP got you garbage points (15ish or worse)
- Mob owner/mob target - in 75 era if you didnt have claim, or the mob was passive or someone else had claim, you got garbage points (5 or less)